code/game/objects/structures/signs/_signs.dm

```diff
@@ -6,8 +6,10 @@
 	layer = SIGN_LAYER
 	max_integrity = 100
 	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
-	var/buildable_sign = 1 //unwrenchable and modifiable
+	var/buildable_sign = 1 //unwrenchable
```

Comment:
```suggestion
	var/buildable_sign = TRUE //unwrenchable
```
This is a boolean, so it should be signalized as such.

Reply:
Fixed.